modules/core/src/test/scala/sangria/execution/InputDocumentMaterializerSpec.scala

```diff
@@ -192,13 +192,14 @@ class InputDocumentMaterializerSpec extends AnyWordSpec with Matchers with Strin
             }]
           }
         """)
+        .get
 
       val vars = scalaInput(
         Map(
           "comm" -> "from variable"
         ))
 
-      InputDocumentMaterializer.to(document, ArticleType, vars) should be(
```

Comment:
This is the sort of implicits magic that I don't like to see. `to()` throws an exception? instead of returning a `Try`? Oh, that's because of the import 200 lines up. One reason that I don't like `DeliveryScheme`.